CHANGELOG.adoc

```diff
@@ -15,6 +15,7 @@
 
 https://github.com/oxidecomputer/dropshot/compare/v0.8.0\...HEAD[Full list of commits]
 
+* https://github.com/oxidecomputer/dropshot/pull/502[#502] Dropshot exposes a `refresh_tls` method to update the TLS certificates being used by a running server.
```

Comment:
I may be mistaken, but I think this is the first breaking change in this list. If that's the case, can you please create two sections as below?

Reply:
Sure, updated in https://github.com/oxidecomputer/dropshot/pull/502/commits/219ff7af4890eed9227d1cb9d5cb9931a5ee16d0